sys/include/net/gnrc/rpl.h

```diff
@@ -458,6 +463,16 @@ gnrc_rpl_dodag_t *gnrc_rpl_root_dodag_init(uint8_t instance_id, ipv6_addr_t *dod
  * @param[in] dodag_id          Id of the DODAG
  */
 void gnrc_rpl_send(gnrc_pktsnip_t *pkt, ipv6_addr_t *src, ipv6_addr_t *dst, ipv6_addr_t *dodag_id);
+
+/**
+ * @brief Get a local or global instance id
+ *
+ * @param[in] local             flag to indicate whether a local or global instance id is requested
+ *
+ * @return  Local instance id, if @p local is true
+ * @return  Global instance id, otherwise.
+ */
+uint8_t gnrc_rpl_get_instance_id(bool local);
```

Comment:
I don't know but the `_get_` in the function name confused me a bit, I misinterpreted it with a kind of getter.
I would vote to change the part to `_gen_` to explicit state that this call generates a new instance ID.